pkg/resources/rdma/genericRdma.go

```diff
@@ -0,0 +1,26 @@
+package rdma
+
+import (
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1"
+
+	"github.com/intel/sriov-network-device-plugin/pkg/types"
+)
+
+type genericRdmaSpec struct {
```

Comment:
Is it possible to detect the rdma mode for a particular device and add a field in this RdmaSpec? for example, whether it's in RoCE, IB or iWARP mode.